pkgs/development/python-modules/frida-python/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  pkgs,
+  typing-extensions,
+}: let
+  pname = "frida-python";
```

Comment:
why are those attrs outside?

Reply:
version needs to be outside because devkit uses it.